### ISSUE TITLE
GM stuck in jail

### ISSUE THREAD

**Issue by Kielbasa** [LegionDark/Issues#1699]
**Date & Time**: 12/05/2017

**FFXI Client Version (use `/ver`)**: 30171103_1

**Server's Expected Client Version matches, yes/no? (use `!ecv`)**: 30171103_1

**Character Name**: Kielbasa

**Nation**: Sandy

**Job(level)/Sub Job(level)**: Thief99/Dancer49

**NPC or Monster or item Name**: 

**Zone name**: Mordion

**Coordinates (use `!where`)**: Jail

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: Memnoch

**Steps To Reproduce / any other info**: Just need to be set free

**Comment by Kielbasa** [comment]
Just need to be let out of jail please xD deleted debuff addon

**Comment by TeoTwawki** [comment]
_**generic copy paste reply:**_
This happens from using tools or addons that try to strip off status effects that you normally cannot.

Each time that happens the code will re-jail. Even if you are already in jail. Please disable any such tools and remove them and any windower or ashita scripts or macros that would attempt to automatically remove such effects (like weakness) so that you don't auto-jail yourself again, and I will release you at my earliest convenience. And hopefully I never see you jail yourself again.


tl;dr
be with you when I get a chance

**Comment by EpicTaru** [comment]
Released.
